src/tasks/level13.ts

```diff
@@ -527,20 +527,15 @@ export const TowerQuest: Quest = {
       },
       completed: () => step("questL13Final") > 10,
       do: $location`Tower Level 5`,
-      outfit: () => {
-        if (have($item`unwrapped knock-off retro superhero cape`))
-          return {
-            equip: $items`unwrapped knock-off retro superhero cape`,
-            modes: { retrocape: ["heck", "hold"] },
-          };
-        else if (have($item`attorney's badge`))
-          return {
-            modifier: "HP",
-            equip: $items`attorney's badge`,
-            avoid: $items`extra-wide head candle`,
-          };
-        else return { modifier: "HP", avoid: $items`extra-wide head candle` };
-      },
+      outfit: () => ({
+        equip: $items`unwrapped knock-off retro superhero cape, Jurassic Parka, attorney's badge`,
+        modes: {
+          modifier: "HP",
```

Comment:
The keys "modifier" and "avoid" should be outside of "modes" here, at the same level as "equip" above

Reply:
Oops – thanks for catching. Fixed